src/Service/Controllers/RestController.cs

```diff
@@ -231,6 +277,12 @@ private async Task<IActionResult> HandleOperation(
                     HttpContextExtensions.GetLoggerCorrelationId(HttpContext));
 
                 Response.StatusCode = (int)ex.StatusCode;
+                if (activity is not null)
+                {
+                    activity.TrackRestControllerActivityFinishedWithWithException(ex, Response.StatusCode);
+                }
```

Comment:
Can't this be simplified as:

```suggestion
                activity?.TrackRestControllerActivityFinishedWithWithException(ex, Response.StatusCode);
```